src/index.tsx

```diff
@@ -15,6 +15,10 @@ import './locales/i18n'
 Axios.defaults.baseURL = process.env.REACT_APP_API
 const queryClient = new QueryClient({})
 
+import { register } from 'swiper/element/swiper-element'
+// register Swiper custom elements
+register()
```

Comment:
How come we aren't using 'swiper/react'?

Reply:
ah, i see, "Swiper React components will likely to be removed in future versions. It is recommended to migrate to Swiper Element instead."